clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsResponse.java

```diff
@@ -271,4 +273,48 @@ public static ApiVersion toApiVersion(ApiKeys apiKey) {
             .setMinVersion(apiKey.oldestVersion())
             .setMaxVersion(apiKey.latestVersion());
     }
+
+    public static ApiVersionsResponse apiVersionsResponse(
+        Integer throttleTimeMs,
```

Comment:
This is a small thing, I guess, but why `Integer` and not `int`? This cannot be null, right?